src/Layout/Schema/index.style.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import '../../index.css';
+
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
You don't need this anymore after you make more that one export

Reply:
@95Rawan 
Done!